parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -70,6 +70,10 @@ configureSecrets(){
     echo "${ETCD_PEER_CERT}" | base64 --decode > "${ETCD_PEER_CERTIFICATE_PATH}"
 }
 
+configPrivateClusterHosts() {
```

Comment:
can we put this into the {{- if EnableHostsConfigAgent}} check?

Reply:
done